src/main/java/org/jabref/gui/actions/StandardActions.java

```diff
@@ -36,6 +36,8 @@ public enum StandardActions implements Action {
     EXTRACT_FILE_REFERENCES_OFFLINE(Localization.lang("Extract references from file (offline)"), IconTheme.JabRefIcons.FILE_STAR),
     OPEN_URL(Localization.lang("Open URL or DOI"), IconTheme.JabRefIcons.WWW, KeyBinding.OPEN_URL_OR_DOI),
     SEARCH_SHORTSCIENCE(Localization.lang("Search ShortScience")),
+    ADD_GROUP_FROM_SELECTION(Localization.lang("Add group from selection")),
```

Comment:
This can be removed? -- (I don't find the link to the comment I gave, maybe you remember the screenshot that "All entries" always have the context menu with "subgroups", not" groups")